### PR TITLE
Extend event interface to allow certain events like NewYear to trigger immediately

### PR DESCRIPTION
**Pull request by mdrie**
I started again writing laws and noticed that testing is not really possibly, because NewYear is not triggered often enough.

Also note the previous commit on the "main" branch. Before that, the proposed laws were not updated correctly and eventually there were no laws to select. I commited that on "main", because it was sooo small.

### CODE REVIEW

**Comment by mdrie** [comment]
Closes #101

File: src/events/EventsTypes.ts [hunk 1]
```diff
@@ -1,17 +1,47 @@
...
   internals?: Internals // TODO #79: Make mandatory
 }
+
+export const eventProbs = {
+  newYear: 2,
+  won: 3,
+  broke: 4,
+  lostElection: 5,
+  co2BudgetUsed: 6,
+}
```
Comment (jschirrmacher) [inline src/events/EventsTypes.ts:47]: I understand that like that: The `eventProbs` are actually a priority. The `newYear` event is less significant as the fact that the `co2BudgetUsed`, so its value is less than the other one.
If that impression is correct, we should rename the variable into something like `eventPriorities`, to make the meaning more clear.

Comment (jschirrmacher) [reply src/events/EventsTypes.ts:47]: Maybe we could even use the event IDs to make the reference more clear.

File: src/events/NewYear.ts [hunk 2]
```diff
@@ -21,6 +22,9 @@ export default defineEvent({
     if (numOfLaws < 3) {
       return 0
     }
+    if (numOfLaws > 4) {
```
Comment (jschirrmacher) [inline src/events/NewYear.ts:25]: This is not the inverse condition and contradicts the probability calculation below which counts 3 to 5 events to make this event possible.

File: src/events/EventsTypes.ts (hunk 1 above)
Comment (mdrie) [reply src/events/EventsTypes.ts:47]: The design is not optimal, there. These are special values for the event probabilities. Maybe `specialEventProbs`?

I am not at all convinced of the design to use a number return value of a function `probability()` and devide its range in several areas with completely different meaning, but i wanted to get it running fast to get back to the laws...

File: src/events/NewYear.ts (hunk 2 above)
Comment (mdrie) [reply src/events/NewYear.ts:25]: The idea is that after 5 laws the event shall happen for certain. Maybe `numOfLaws >= 5` would have made that more apparent?

File: src/events/EventsTypes.ts (hunk 1 above)
Comment (mdrie) [reply src/events/EventsTypes.ts:47]: Event IDs would be nice, but are unavailable at this point. We could still use them here "by hand" to make the correspondence more apparent.

**Comment by mdrie** [comment]
I am not sure if it is ok to merge now.

File: src/events/EventsTypes.ts (hunk 1 above)
Comment (jschirrmacher) [reply src/events/EventsTypes.ts:59]: Ok, lets keep it like that.

File: src/events/NewYear.ts (hunk 2 above)
Comment (jschirrmacher) [reply src/events/NewYear.ts]: Ah, ok, I didn't get it right.

**Review by jschirrmacher — APPROVED** [review]

**Pull request merged**